share/completions/climate.fish

```diff
@@ -0,0 +1,82 @@
+complete -f -n '__fish_climate_already_has_argument' -c climate -a uninstall -d 'uninstall your climate installi'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a version -d 'Show climate version'
+
+# Info
+complete -f -n '__fish_climate_already_has_argument' -c climate -a weather -d 'Get the weather'
+
+# General
+complete -f -n '__fish_climate_already_has_argument' -c climate -a battery -d 'Display remaining battery'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a sleep -d 'Display remaining battery'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a lock -d 'Lock computer'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a shutdown -d 'Shutdown the computer'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a restart -d 'Restart the computer'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a time -d 'Show the time'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a clock -d 'Put a console clock in the top right corner'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a countdown -d 'A countdown timer'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a stopwatch -d 'A stopwatch'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a ix -d 'Pipe output to ix.io'
+
+# Files
+complete -n '__fish_climate_already_has_argument' -c climate -a biggest-files -d 'Find the biggest files recursively'
+complete -n '__fish_climate_already_has_argument' -c climate -a biggest-dirs -d 'Find the biggest directories'
+complete -n '__fish_climate_already_has_argument' -c climate -a dir-size -d 'Find directory size'
+complete -n '__fish_climate_already_has_argument' -c climate -a remove-empty-dirs -d 'Remove empty directories'
+complete -n '__fish_climate_already_has_argument' -c climate -a extract -d 'Extract any given archive'
+complete -n '__fish_climate_already_has_argument' -c climate -a find-duplicates -d 'Report duplicate files in a directory'
+complete -n '__fish_climate_already_has_argument' -c climate -a count -d 'Count the number of occurences'
+complete -n '__fish_climate_already_has_argument' -c climate -a monitor -d 'Monitor file for changes'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a grep -d 'Search for the given pattern recursively'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a replace -d 'Replace all occurences'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a ramfs -d 'Create a ramfs of size (in MB) at path'
+
+# Network
+complete -f -n '__fish_climate_already_has_argument' -c climate -a speedtest -d 'Test your network speed'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a local-ip -d 'Retrieve your local ip address'
+complete -f -n '__fish_climate_already_has_argument' -c climate -a is-online -d 'Verify if you''re online'
```

Comment:
Here, fish concatenates the two strings. It's useful when you want to expand variables:
```fish
> set var 'Hi there!'
> echo 'Did you say "'$var'"?'
```
```
Did you say "Hi there!"?
```

It works like a charm with lists as well:
```fish
> set var 'Hi there!' 'Hello world!'
> echo 'Did you say "'$var'"?'
```
```
Did you say "Hi there!"? Did you say "Hello world!"?
```